### PR TITLE
4486 - add "address removed" in tooltip when user removes address

### PR DESCRIPTION
**Pull request by Miaplacidus**
Add "Address removed" message to tooltip that appears when hovering over user avatar present in header of thread/thread preview. 

## Link to Issue
Closes: #4486

## Description of Changes
- add "Address removed" text to tooltip when address not present

### CODE REVIEW

**Review by mzparacha — COMMENTED** [review]
I noticed 2 other instanced of `redactedAddress` in the `userPopover` var -- maybe we should update them too since they are part of tooltip?

**Review by lzach83 — APPROVED** [review]
LGTM

File: packages/commonwealth/client/scripts/views/components/user/user.tsx [hunk 1]
```diff
@@ -204,9 +204,9 @@ export const User = ({
               </Link>
             )}
           </div>
-          {profile?.address && (
-            <div className="user-address">{redactedAddress}</div>
-          )}
+          <div className="user-address">
+            {profile?.address ? redactedAddress : 'Address removed'}
+          </div>
```
Comment (mhagel) [inline packages/commonwealth/client/scripts/views/components/user/user.tsx:209]: Is this the expected look? It says "Address removed," but the truncated address is still visible where the Profile name would be.

Screenshot 2023-11-08 at 12 06 13 PM

**Review by mhagel — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**